spring-boot-starters/spring-boot-starter-data-geode/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starters</artifactId>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-boot-starter-data-geode</artifactId>
+	<name>Spring Boot Data Geode Starter</name>
+	<description>
+		The Spring Boot Data Geode Starter including Spring Data GemFire with support for Apache Geode.
+	</description>
+	<url>http://projects.spring.io/spring-boot/</url>
+	<organization>
+		<name>Pivotal Software, Inc.</name>
+		<url>http://www.spring.io</url>
+	</organization>
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-gemfire</artifactId>
+			<version>${spring-data-geode.version}</version>
```

Comment:
The only option then is to create a new, top-level SD project by pulling the `apache-geode` branch out into a new source code repository in order to release a new SD module (i.e. **Spring Data Geode**) on which the _Spring Boot Starter_ will be based

A long time will pass before there exists a version of _Pivotal GemFire_ based on _Apache Geode_ where potentially SDG can support both _GemFire_ and _Geode_ simultaneously in a single release (if ever).

Without this, there is simply no way to add early access (preview-style) support for _Apache Geode_ in _Spring Boot_.